changelog_generator/repository_manager.py

```diff
@@ -17,19 +22,42 @@
 
 def is_tag_version(tag: str) -> bool:
     res = tag_version_re.match(tag)
-    if not res:
-        return False
-    return not res.group("rc")
+    return res and not res.group("rc")
 
 
-def get_tag_value(tag: str) -> int:
+def get_tag_value(tag: str) -> Tuple:
+    """This splits a tag in a tuple of ints. This makes it straightforward to compare two tags."""
```

Comment:
Indeed comparing tuple is easier, someday I should learn python :)
Comment is useless and function should be renamed to `get_semver_from_tag`